src/app/features/mushaf/components/mushaf-page/mushaf-page.component.less

```
@@ -0,0 +1,85 @@
+.mushaf-page {
+  &__svg {
+    width: 100%;
+    max-width: 720px;
+    margin-inline: auto;
+
+    &--hidden {
+      display: none;
+    }
+
+    // Injected SVG scales to the container width.
+    ::ng-deep svg {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)